AUTHORS

```diff
@@ -1,4 +1,5 @@
 Aaron Lehmann <aaron.lehmann@docker.com>
+Aaron Vinson <avinson.public@gmail.com>
```

Comment:
Please do not include AUTHORS updates in PRs. These are updated at release time.